plugins/search-backend-module-elasticsearch/src/engines/ElasticSearchSearchEngine.ts

```diff
@@ -452,6 +454,24 @@ export async function createElasticSearchClientOptions(
         : {}),
     };
   }
+  if (config.getOptionalString('provider') === 'opensearch') {
+    const authConfig = config.getConfig('auth');
+    return {
+      provider: 'opensearch',
+      auth: {
+        username: authConfig.getString('username'),
+        password: authConfig.getString('password'),
+      },
+      ...(sslConfig
```

Comment:
It is missing the node config...

Reply:
Solved on commit 9de6464aee55e0b9bc978292941d9d87f13baead